File: Models/Bioskop.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace Models
{
    public class Bioskop
    {
        [Key]
        public int ID { get; set; }

        [RegularExpression(@"\w+")]
        [Required]
        [MaxLength(50)]
        public string Naziv { get; set; }

        public List<Sala> Sale { get; set; }
    }
}

File: Client/Bioskop.js
export class Bioskop{

    constructor(id, naziv){
        this.id = id;
        this.naziv = naziv;
        this.sale = [];
    }

    DodajSalu(sala){
        if(this.id === sala.bioskopid){
            this.sale.push(sala);
        }
    }
}